### ISSUE TITLE
Pyramid `inside` option not working

### ISSUE THREAD

**Issue by piotr-m-jurek** [highcharts/highcharts#10036]
#### tl;dr 
Setting `plotOptions.pyramid.dataLabels.inside` to true works not exactly as intended. The font changes (sometimes), but nothing more.

#### Expected behaviour

Labels are inside the chart, not outside, connected with connector

#### Actual behaviour
Labels are outside the chart, with connectors, some of them change font color and outline
![image](https://user-images.githubusercontent.com/11314344/52109423-b54a2680-25fd-11e9-8a58-a2d77e333ccc.png)

#### Live demo with steps to reproduce

https://jsfiddle.net/8tuvpfc4/2/

#### Product version

 Highcharts JS v7.0.2 (2019-01-17)

#### Affected browser(s)
Firefox, Chrome, Opera, Safari

**Comment by sebastianbochan** [comment]
Hi @piotr-m-jurek,
Thank you for reporting. The color of font is contrasted and based on shape color. If you have i.e black shape, color is white to keep readability.

The issue is caused by positive distance, set the param as negative.

**Demo:**
- https://jsfiddle.net/BlackLabel/1957Ldvz/

Let me know if it is your goal, if not, please attach a mockup.

**Comment by piotr-m-jurek** [comment]
Hi @sebastianbochan thanks for quick response, 
You are right about the font color, thank you!

What is the purpose of settings like `inside` then? The description isn't really clear at this point

**Comment by sebastianbochan** [comment]
@TorsteinHonsi should we consider it as a bug in functionality or docs?

**Comment by TorsteinHonsi** [comment]
Strictly speaking it's a documentation issue because `inside` isn't implemented for pie series and derivatives. I've removed it from the API now.

That said, it would makes total sense for funnels and pyramids, so I'll tag this as an enhancement.

**Comment by piotr-m-jurek** [comment]
Is it possible this issue will be resolved any time soon?

**Comment by wchmiel** [comment]
I will check it and try to make as soon as possible. However, I'm not able to estimate time.

**Comment by piotr-m-jurek** [comment]
> I will check it and try to make as soon as possible. However, I'm not able to estimate time.

thank you!

**Comment by KacperMadej** [comment]
Closing as resolved as the feature is implemented.
